config/default.yml

```diff
@@ -519,8 +519,10 @@ Style/HashSyntax:
     - ruby19
     - ruby19_no_mixed_keys
     - hash_rockets
-# Force hashes that have a symbol value to use hash rockets
+  # Force hashes that have a symbol value to use hash rockets
   UseHashRocketsWithSymbolValues: false
+  # Do not suggest { a?: 1 } over { :a? => 1 } in ruby19 style
+  AllowHashRocketsWithSymbolValuesEndingInNonAlnum: true
```

Comment:
That's a mouthful. :-) Probably we can shorten it a bit.

I'm also not sure we should set it to true by default.